### PR TITLE
chore: add option to skip properties for a node

### PR DESCRIPTION
**Pull request by yashmehrotra**
@losisin Would you interested in merging a feature like this ?

Usecase:
```
labels:
  default:
    app.k8s.io:/name: abc
    app.k8s.io/namespace: def
```

Ideally, in this case I would only want labels.default to be an object, and not have any  properties as the yaml is completely a user input blob, and can have any properties

So adding `skipProperties: true` here would not those in schema

### CODE REVIEW

**Review by losisin — DISMISSED** [review]

**Comment by yashmehrotra** [comment]
@losisin Thanks for approving the changes, I think the Test CI workflow is waiting for your approval

**Comment by losisin** [comment]
I like the initial better. If you can add test cases I can merge it.

**Comment by yashmehrotra** [comment]
@losisin Have added tests and made sure coverage is at 100%

Let me know if anything else is required

**Review by losisin — APPROVED** [review]

**Pull request merged**
